code/modules/medical/genetics/geneticsItems.dm

```diff
@@ -189,3 +175,78 @@
 		else
 			boutput(user, "<span class='alert'>You stab [M], but nothing happens.</span>")
 		return
+
+/obj/item/dna_scrambler
+	name = "dna scrambler"
+	desc = "An illegal retroviral genetic serum designed to randomize the user's identity, store it, and apply it later."
+	icon = 'icons/obj/syringe.dmi'
+	inhand_image_icon = 'icons/mob/inhand/hand_medical.dmi'
+	item_state = "syringe_0"
+	icon_state = "dna_scrambler_1"
+	force = 3
+	throwforce = 3
+	w_class = W_CLASS_SMALL
+	var/use_mode = "copy"
+	var/datum/bioHolder/bioHolder = new/datum/bioHolder
+	var/stored_name
+	contraband = 2
+
+	attack(mob/M as mob, mob/user as mob)
```

Comment:
```suggestion
	attack(mob/M, mob/user)
```

These don't do anything for procs that aren't verbs- they still show up in a ton of places but that's just because nobody has cared to remove them. :waa: